### PR TITLE
Exclude LambdaLoadTest_ConcurrentScavenge from ppc64le_linux_xl

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Exclude for : https://github.com/eclipse/openj9/issues/6475

FYI @pshipton 

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Test : Test_grinder/job/Grinder/5980/console

File: system/lambdaLoadTest/playlist.xml [hunk 1]
```diff
@@ -136,7 +136,7 @@
 			<impl>openj9</impl>
 			<impl>ibm</impl>
 		</impls>
-		<platformRequirements>bits.64,^arch.arm</platformRequirements>
+		<platformRequirements>bits.64,^arch.arm,^spec.linux_ppc-64_le</platformRequirements>
```
Comment (pshipton) [inline system/lambdaLoadTest/playlist.xml]: Is spec.linux_ppc-64_le only going to exclude XL and not compressed refs?
Also it needs to be excluded for xlinux XL as well.

Comment (Mesbah-Alam) [reply system/lambdaLoadTest/playlist.xml]: > Is spec.linux_ppc-64_le only going to exclude XL and not compressed refs?

Yes : https://github.com/AdoptOpenJDK/openjdk-tests/blob/28a458bbc492b8d287f78097d4d6ba041e4b7787/buildenv/jenkins/openjdk_tests#L45

Comment (Mesbah-Alam) [reply system/lambdaLoadTest/playlist.xml]: Excluded xLinux XL as well. Please refresh PR to check. 

Test: /Test_grinder/job/Grinder/5990/console

**Comment by llxia** [comment]
Please also run a Grinder for an enabled spec. Thanks

**Comment by Mesbah-Alam** [comment]
@llxia : Sure, it passes : Test_grinder/job/Grinder/5993/console

**Pull request merged**